vector/build.gradle

```diff
@@ -247,13 +249,55 @@ android {
         fdroid {
             dimension "store"
 
-            versionName "${versionMajor}.${versionMinor}.${versionPatch}${getFdroidVersionSuffix()}"
-
             resValue "bool", "isGplay", "false"
             buildConfigField "boolean", "ALLOW_FCM_USE", "false"
             buildConfigField "String", "SHORT_FLAVOR_DESCRIPTION", "\"F\""
             buildConfigField "String", "FLAVOR_DESCRIPTION", "\"FDroid\""
         }
+
+        withvoip {
+            dimension "voip"
+        }
+
+        withoutvoip {
+            dimension "voip"
+        }
+
+        withpinning {
+            dimension "pinning"
+        }
+
+        withoutpinning {
+            dimension "pinning"
+        }
+
+        // Note: we cannot use reserved word protected, so typo the word
+        secure {
+            dimension "target"
+
+            applicationId "fr.gouv.tchap"
+            versionName "${versionMajor}.${versionMinor}.${versionPatch}"
```

Comment:
build on a not tagged commit will have a "-dev" suffix
Please create our own method by taking example on the existing "getFdroidVersionSuffix()"